pkg/bgpv1/test/testdata/svc-gateway-dummy-endpoints.txtar

```diff
@@ -0,0 +1,149 @@
+#! --test-peering-ips=10.99.4.101,10.99.4.102
```

Comment:
Please use unique IPs for this test (each component test needs to use unique test peering IPs, so that they can run in parallel without affecting each other)